utils/generate-rubocop-rules/src/main/java/org/sonarsource/slang/externalreport/rubocop/RuboCopRuleDefinition.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.slang.externalreport.rubocop;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.text.StringEscapeUtils;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class RuboCopRuleDefinition {
+
+  private static final Path RULES_FILE = Paths.get("sonar-ruby-plugin", "src", "main", "resources",
+    "org", "sonar", "l10n", "ruby", "rules", "rubocop", "rules.json");
+
+  private static final Path RUBOCOP_HELP = Paths.get("utils", "generate-rubocop-rules",
+    "src", "main", "resources", "rubocop.yml");
+
+  public static void main(String[] args) throws IOException {
+    String rules = RuboCopRuleDefinitionGenerator.generateRuleDefinitionJson(resolve(RUBOCOP_HELP));
+    Files.write(resolve(RULES_FILE), rules.getBytes(UTF_8));
+  }
+
+  private static Path resolve(Path path) throws IOException {
```

Comment:
Why do we need that?

Reply:
It's mainly to resolve "sonar-ruby-plugin" folder, without forcing you to execute `mvn exec:java` in the root directory of slang repo.